modules/ROOT/pages/custom-policy-4-reference.adoc

```diff
@@ -170,11 +169,11 @@ Then, operations exposed by the dependency are available to be used within polic
 </mule>
 ----
 
-NOTE: It is not possible to use Java Module from policies. Java Module requires that a policy exports the Java classes it has to use, and policies are not allowed to export packages nor resources.
+NOTE: It is not possible to use a Java Module from policies. A Java Module requires that a policy exports the Java classes it has to use, and policies are not allowed to export packages nor resources.
```

Comment:
The second reference to the Java Module:
>  A Java Module requires that a policy exports the Java classes it has to use (...)

Should be:
> The Java Module requires that a policy exports the Java classes it has to use (...)

Reply:
Hi @fermujica !  I'll be glad to help, but I'm not quite sure how to use the changes you've requested to my changes?  I resolved one conversation, but I'm not sure that was the right way to add your changes?  Let me know how I can help!

Reply:
You should be able to edit your changes by going to the **Files changed** tab, and click the **Change this File using the online editor** button. 
<img width="882" alt="screen shot 2018-12-17 at 2 16 37 pm" src="https://user-images.githubusercontent.com/16763154/50103586-bcaca080-0206-11e9-85df-e2c9467b593d.png">


After you applied all the changes I requested, commit them and they'll be added to this pull request.